projects/adv7511/readme.txt

```diff
@@ -0,0 +1,9 @@
+Additional required source files:
+cp ../../common_drivers/dmac_core/dmac_core.c .
+cp ../../common_drivers/dmac_core/dmac_core.h .
+cp ../../common_drivers/platform_drivers/platform_drivers.c .
+cp ../../common_drivers/platform_drivers/platform_drivers.h .
+
+The pregram also needs the ADV6511 HDMI transmitter library API source code downloaded from:
```

Comment:
I would rename the file to README for consistency with the rest of the reame files already available on the no-OS repository and I will fix the "pregram" typo. :)